Rubberduck.Core/UI/Command/AddTestModuleCommand.cs

```diff
@@ -121,18 +121,16 @@ private string DeclarationFormatFor(string declarationFormat, string type, IUnit
 
         private IVBProject GetProject()
         {
-            using (var activeProject = _vbe.ActiveVBProject)
+            var activeProject = _vbe.ActiveVBProject;
+            if (!activeProject.IsWrappingNullReference)
             {
-                if (!activeProject.IsWrappingNullReference)
-                {
-                    return activeProject;
-                }
+                return activeProject;
             }
 
             using (var projects = _vbe.VBProjects)
             {
                 return projects.Count == 1
-                    ? projects[1]
+                    ? projects[1] // because VBA-Side indexing
```

Comment:
#nitpick: not really a VBA thing. It's a *Microsoft* thing. ;) VBA on itself will support zero-based array; Microsoft went out of their way to allow for one-based collections/arrays in *some* of objects because you know, noobs....

Reply:
Wouldn't `.First` be less ambiguous then? The instance would be yielded from the enumerator..